database/mysql/mysql.go

```diff
@@ -253,16 +261,36 @@ func (m *Mysql) Open(url string) (database.Driver, error) {
 		}
 	}
 
+	metadataLockTimeoutParam := customParams["x-metadata-lock-timeout"]
+	metadataLockTimeout := uint64(0)
+	if metadataLockTimeoutParam != "" {
+		metadataLockTimeout, err = strconv.ParseUint(metadataLockTimeoutParam, 10, 32)
```

Comment:
Can we declare the 32 as a const called `uintBitSize` or similar?

Is it possible that this value will change in the future?

Reply:
It's more just the largest number we expect in the input string. The actual range of numbers that are sensible to put here are somewhere between like 0 and 60. We could probably get away with a uint8 and no user would ever exceed it. 

The parse function outputs a uint64 anyways. All the uint sizes are very much me going "i don't care" and trying not to fight the type checker too hard.